### ISSUE TITLE
TetView Process persists after window and Matlab close

### ISSUE THREAD

**Issue by siboles** [gibbonCode/GIBBON#13]
@Kevin-Mattheus-Moerman
Windows testing exhibited this memory leak behaviour for the TetView process. The tetgen on my linux machine does not spawn a tetview window. Perhaps this is unique to the Windows TetView/tetgen binary. Just in case this is not a 3rd party software issue, figured it is worth mentioning.

**Comment by Kevin-Mattheus-Moerman** [comment]
Thanks for catching this. Yes Tetview uses some old libraries and I have never been able to get it to run on Ubuntu. I decided to remove it and phase out TetView based mesh viewing in GIBBON. To replace it I've added `meshView`, a new function to view meshes. I've replaced all occurrences of `runTetView` with `meshView`, e.g. `HELP_runTetGen` now uses it. 
If users do use the `runTetView` function they will receive a warning that it will be removed and that `meshView` should be used instead.
